cornac/eval_methods/next_basket_recommendation_evaluation.py

```diff
@@ -0,0 +1,743 @@
+# Copyright 2023 The Cornac Authors. All Rights Reserved.
```

Comment:
Please also let this inherits from `BaseMethod` and see how we can minimize redundancy.

Reply:
If the difference is only in the `ranking_eval()` function, we only need to overwrite the `_eval()`. Also, can we consider shortening this to `NextBasketEvaluation`? (just a suggestion)